Dockerfile

```diff
@@ -42,14 +42,17 @@ COPY --from=builder /build/cni-download /usr/libexec/cni
 COPY build/image-assets/iptables-wrapper-installer.sh /
 # This is necessary because of the bug reported here: https://github.com/flannel-io/flannel/pull/1340/files
 # Basically even under QEMU emulation, it still doesn't have an ARM kernel in-play which means that calls to
-# iptables-nft will fail in the build process. The sanity check here only makes sure that we are not using
-# iptables-1.8.0-1.8.2. For now we'll manage that on our own.
-RUN /iptables-wrapper-installer.sh --no-sanity-check
-
-
-# Since alpine image doesn't contain /etc/nsswitch.conf, the hosts in /etc/hosts (e.g. localhost)
-# cannot be used. So manually add /etc/nsswitch.conf to work around this issue.
-RUN echo "hosts: files dns" > /etc/nsswitch.conf
```

Comment:
Do we still need this?

Reply:
No, we shouldn't need it, like @elchenberg mentioned it is now default this way via the Alpine base image.